components/RestaurantDetail.js

```diff
@@ -1,70 +1,85 @@
-          </div>
-          <div className='inline mr-5 border-solid border-gray-500 border-r pr-3'>{(Math.floor(Math.random() * 5000)).toLocaleString('en-US')} ratings</div>
-          <div className='inline'>{data.priceRange}</div>
+      <div className='pl-3 h-[185px] bg-[#1F2427] flex flex-col'>
+        <div className='font-bold text-3xl tracking-wide pt-2.5 pb-1 mt-[-4px] mb-[5px] text-white'>{restaurant.restaurantName}</div>
+
+        <div className='pr-3 border-solid border-gray-500 border-r flex text-white'>
+          <p> {restaurant.cuisine} Restaurant </p>
+          <p className='text-white text-2xl ml-[10px] mt-[-4px]'>|</p>
+          <p className='ml-[10px] mt-[3px] text-sm'> {Math.floor(Math.random()*(4000-2000+1)+2000)} </p>
+          <p className='ml-[5px] mt-[3px] text-sm'> ratings </p>
+          <p className='text-white text-2xl ml-[10px] mt-[-4px]'>|</p>
+          <svg className='ml-[5px] mt-[8px] pl-1 pb-1 fill-[#A38022]'  width="15" height="15" viewBox="0 0 24 24"><path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z"/></svg>
+          <p className='ml-[10px] mt-[3px] text-sm'> {Number.parseFloat(Math.random()*(4.50-3.30)+(3.30)).toFixed(2)} </p>
+        </div>
+
+        <div className="pr-3 h-[30px] mt-[5px] flex">
+          <div className="text-white text-center  bg-[#657841] ml-[5px] mt-[3px] w-[45px] h-[24px] rounded-[25px] text-[14px]">$$$</div>
+          <div className="text-white text-center bg-[#3475A9] ml-[13px] mt-[3px] w-[55px] h-[24px] rounded-[25px] text-[14px]">Fast</div>
+          <div className="text-white text-center bg-[#657841] ml-[16px] mt-[3px] w-[75px] h-[24px] rounded-[25px] text-[14px]">12 miles</div>
+        </div>
+
+        <div className="pr-3 h-[60px] mt-[5px] text-white">
+        <p> {restaurant.description} </p>
+
         </div>
+
+      </div>
+
+      <div className="h-[4px] w-full  bg-[#38474F]">
       </div>
-      <ToggleSwitch />
-      <div className='bottom-20 mt-5 pl-3'>
-        <div className='font-bold text-lg tracking-wide border-b pb-4 mr-3'>Full Menu</div>
-        {data.dishes.map((dish, index) => {
-          return (
-            <div className='flex justify-between mt-6 border-b pb-4 mr-4 cursor-pointer' key={index}>
-              <div className='w-9/12'>
-                <div className='font-bold'>{dish.name}</div>
-                <div>{`This shit's fir${'eee'.repeat(index + 1)}`}</div>
-                <div>{'$' + parseFloat(Math.random() * (30 - 10) + 10).toFixed(2)}</div>
-              </div>
-              <img className='w-20 h-20 rounded-md object-cover object-center' src={dish.photoURL}></img>
-            </div>
-          );
-        })}
+
+      <div className='bottom-20 pl-3 h-[70px] flex items-center bg-[#1F2427]'>
+        <div className='font-bold text-2xl text-white mr-3'>Menu</div>
       </div>
+
+      <div className='pl-3 pr-3 w-full bg-[#1F2427]'>
```

Comment:
We have a defined color for the background, so it should be using bg-dark-jungle-green rather than this hard-coded value.

Reply:
> 

Clickhandler was originally from Chris. 

> **Nitpicks** I'm seeing a comment on line 32 of `components/DishCard_PageJ.js`
> 
**fixed**
> In `components/RestaurantCard.js` I see that `clickHandler` was renamed to `ClickHandler`. Why? It's not a component.
> 
**fixed, this was originally from Chris, i didnt name it.**

> In `components/SelectedRestaurants.js` on line 41 I see a console.log
> 
**fixed** 

> in `pages/Page-J-Restaurant.js` on line 37 I see a console.log
> 
**fixed** 

> in `server/index.js` I'm seeing a commit made to it but no changes? What was the purpose for creating a commit here?
> 
**Not sure, I added untracked files to the staging area, not sure how it got there**

> I see a console.log in `server/routes/restaurantRoutes.js` on line 36
> 
**fixed** 

> No comments on the back-end. Not tested.